releases/release-4.0.5.md

```diff
@@ -0,0 +1,137 @@
+    - Fix issue `pre_split_regions` table option doesn't work in the partition table. [#18837](https://github.com/pingcap/tidb/pull/18837)
+    - Fixed an issue that could cause a large transaction to be terminated prematurely. [#18813](https://github.com/pingcap/tidb/pull/18813)
+    - Fix incorrect collator when `getSignatureByPB` and remove unnecessary recover [#18735](https://github.com/pingcap/tidb/pull/18735)
+    - Fix a bug `getAutoIncrementID()` function does not consider the `tidb_snapshot` session variable, this bug may cause dumper tool fail with 'table not exist' error. [#18692](https://github.com/pingcap/tidb/pull/18692)
+    - Fix unknown column error for sql like `select a from t having t.a` [#18434](https://github.com/pingcap/tidb/pull/18434)
+    - Fix a panic on hash partition table when the query condition is that the partition column equals to a big number like 9223372036854775808 [#18186](https://github.com/pingcap/tidb/pull/18186)
+    - Fix the wrong behavior of char function. [#18122](https://github.com/pingcap/tidb/pull/18122)
+    - Ddl: fix admin repair table with range partition expr will parseInt fail [#17988](https://github.com/pingcap/tidb/pull/17988)
+    - Fix wrong behavior of set charset statement [#17289](https://github.com/pingcap/tidb/pull/17289)
+    - Fix a bug caused by the wrong collation setting which leads to the wrong result of collation function. [#17231](https://github.com/pingcap/tidb/pull/17231)
+    - Fix STR_TO_DATE's handling for format token '%r', '%h' [#18727](https://github.com/pingcap/tidb/pull/18727)
+    - Fix issues of TiDB version information formation doesn't consistent with PD/TiKV in cluster_info table. [#18413](https://github.com/pingcap/tidb/pull/18413)
+    - Fix the existence checks for pessimistic transactions. [#19004](https://github.com/pingcap/tidb/pull/19004)
+    - Fix union select for update race. [#19006](https://github.com/pingcap/tidb/pull/19006)
+    - Fix the wrong query result when apply has a child of type PointGet. [#19046](https://github.com/pingcap/tidb/pull/19046)
+
++ TiKV
+
+    - Fix memory leak during scheduling [#8357](https://github.com/tikv/tikv/pull/8357)
+    - Speed up leader election when hibernate region is enabled [#8292](https://github.com/tikv/tikv/pull/8292)
+
++ PD
+
+    - Fix the bug that TSO request may fail at the time of leader changing. [#2666](https://github.com/pingcap/pd/pull/2666)
+    - Fix the issue that when enabling placement rules, sometimes region replicas cannot schedule to optimal [#2720](https://github.com/pingcap/pd/pull/2720)
+    - Fix the bug that balance-leader won't work when placement rule enabled [#2726](https://github.com/pingcap/pd/pull/2726)
+
++ TiFlash
+
+    - Fix the issue that TiFlash cannot start normally after upgrading from an old version if the name of the database or table contains special characters
+    - Fix the issue that TiFlash process can not exit if any exceptions are thrown during initialization
```

Comment:
- Fix the issue that TiFlash process can not exit if any exceptions are thrown during initialization [#953](https://github.com/pingcap/tics/pull/953)